src/lib/menu/menu-trigger.ts

```diff
@@ -140,6 +143,7 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
               @Optional() @Self() private _menuItemInstance: MatMenuItem,
               @Optional() private _dir: Directionality,
               // TODO(crisbeto): make the _focusMonitor required when doing breaking changes.
+              // @deletion-target 6.0.0
```

Comment:
Is deletion target the right tag?

Reply:
That's what we've been using in other places to mark that a parameter should become required. `deletion-target` might not be the best naming for it though.